pypolo/settings.py

```diff
@@ -122,6 +122,21 @@
 # Default number of threads for workers
 NUM_THREADS = 1
 
+# https://poloniex.com/fees/
+FEES = [
```

Comment:
As we may want to support multiple markets down the line, do we want to go ahead and split this into something where we can store multiple fee structures by key/market?